source/MaterialXRenderMsl/MslRenderer.mm

```diff
@@ -124,6 +124,17 @@
     _program->getAttributesList();
 }
 
+void MslRenderer::updateUniform(ShaderPort* uniform)
+{
+
+    if (!uniform)
+    {
+        return;
+    }
+
+    _glProgram->bindUniform(uniform->getVariable(), uniform->getValue());
```

Comment:
Does the `_glProgram` member variable exist on this platform?

Reply:
Looks like a typo. Guessing it should be `_program`.